.github/workflows/reusable_compatibility.yml

```diff
@@ -110,7 +110,7 @@ jobs:
       run: >
         UMF_LOG="level:warning;flush:debug;output:stderr;pid:no" 
         LD_LIBRARY_PATH=${{github.workspace}}/latest_version/build/lib/ 
-        ctest --output-on-failure
+        ctest --output-on-failure -E "umf-mempolicy"
```

Comment:
Why this test has to be disabled? If this is because of a compatibility break, we could merge PRs with a failing compatibility tests workflow since this PR until the next tag. After all, this is a change that breaks compatibility on purpose.

Otherwise, a clear TODO to restore this test after the next tag is created should be placed here.

Reply:
It's not a part of this PR, please focus just on latest commit.